CODE_OF_CONDUCT.md

```diff
@@ -67,10 +67,10 @@ members of the project's leadership.
 
 ## Attribution
 
-This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
-available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 2.1,
```

Comment:
You have to copy-paste the file from not just update the link 😅 https://www.contributor-covenant.org/version/2/1/code_of_conduct/code_of_conduct.md

Reply:
Oh Okay Sorry I misunderstood it. I will make changes  right away.